modules/entity/spec/sorted_state_adapter.spec.ts

```diff
@@ -407,4 +407,36 @@ describe('Sorted State Adapter', () => {
       },
     });
   });
+
+  it('should let you add one entity to the state with setOne()', () => {
+    const withOneEntity = adapter.setOne(TheGreatGatsby, state);
+    expect(withOneEntity).toEqual({
+      ids: [TheGreatGatsby.id],
+      entities: {
+        [TheGreatGatsby.id]: TheGreatGatsby,
+      },
+    });
+  });
+
+  it('should let you replace an entity in the state with setOne()', () => {
+    const withOne = adapter.addOne(
+      TheGreatGatsby,
+      adapter.addOne(AnimalFarm, adapter.addOne(AClockworkOrange, state))
```

Comment:
```suggestion
      state
```

Reply:
I'm not sure if I understand this. The reason I'm adding three books to the collection here is to test that sorted adapter preserves the correct sorting order after a 'setOne' call. If I only have one book in the collection that is not checked anymore. Do you think that is okay?

Reply:
Oh yea, that makes sense! Perhaps rename the `withOne` variable then tho

Reply:
Oops. That's a copy&paste error :(. Should be fixed now.